.github/workflows/ci.yml

```
@@ -12,13 +12,27 @@ jobs:
         with:
           options: "--color --check --verbose"
   tests:
-    name: Run Unit Tests
-    runs-on: ubuntu-latest
+    name: ${{ matrix.name }}
+    runs-on:  ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - {name: Linux39, python: '3.9', os: ubuntu-latest, tox: py39}
+          - {name: Windows39, python: '3.9', os: windows-latest, tox: py39}
+          - {name: Mac39, python: '3.9', os: macos-latest, tox: py39}
```

```suggestion
          - { name: Linux39, python: '3.9', os: ubuntu-latest, tox: py39 }
          - { name: Windows39, python: '3.9', os: windows-latest, tox: py39 }
          - { name: Mac39, python: '3.9', os: macos-latest, tox: py39 }
```
consistency and readability